streams-activitypub-graph/src/test/resources/application.conf

```diff
@@ -1,9 +1,28 @@
+SERVER_BASEURL = "https://mastodon.social"
+NodeinfoApiStaticImplConfig = {
+  version = "1.0"
+  server = {
+    baseUrl = "https://example.com"
+    baseUrl = ${?SERVER_BASEURL}
+    name = "Apache Streams ActivityPub Server"
+    software = "apache-streams-activitypub"
+    version = "0.8.0-SNAPSHOT"
+  },
+  protocols = ["activitypub"]
+  services = {
+    inbound = []
+    outbound = []
+  }
+  openRegistrations = false
+}
 ActivityPubGraphTestSuiteExtensionConfig = {
   fusekiEndpointURI = "http://localhost:13330/"
   testDatasetResource = "doap.ttl"
   testDatasetId = "doap.ttl"
 }
-WebfingerGraphImplConfig = {
+BaseGraphImplConfig = {
   fusekiEndpointURI = "http://localhost:13330/"
   defaultDatasetId = "doap.ttl"
 }
+ProfileGraphImplConfig = ${BaseGraphImplConfig}
+WebfingerGraphImplConfig = ${BaseGraphImplConfig}
```

Comment:
src/test/resources/application.conf should (i believe) not rely on contents of any reference.conf files, b/c its not worth the trouble to wrangle them across modules at test time.  

the shade/war plugin typically merges all reference.conf and application.conf files with append strategy during dist, so perhaps we can add some test cases there to test against a live container running a fully default configuration